app/build.gradle

```diff
@@ -22,38 +23,68 @@ android {
             proguardFiles getDefaultProguardFile('proguard-android-optimize.txt'), 'proguard-rules.pro'
         }
     }
+
+    compileOptions {
+        sourceCompatibility JavaVersion.VERSION_1_8
+        targetCompatibility JavaVersion.VERSION_1_8
+    }
 }
 
 kapt {
     generateStubs = true
 }
 
+ext {
```

Comment:
move it into the buildscript in the project gradle

Reply:
fixed it